README.md

```diff
@@ -183,6 +183,18 @@ This is Day to Day Work Linux Cheatsheet for Software Engineers.
 | Run command2 if command1 is successful     | `<command1> && <command2>`   |
 | Run command2 if command1 is not successful | `<command1> \|\| <command2>` |
 
+## Network Commands
+| Name                                                                           | Command                |
+| -------------------------------------------------------------------------------| -----------------------|
+| Display all network ports and ip address                                       | `ifconfig -a`          |
+| Send echo requrt to test connection                                            | `ping <host>`          |
+| Get who is information for domain                                              | `whois <domain>`       |
+| Get DNS infornmation for domain                                                | `dig <domain>`         |
+| Reverse lookup host                                                            | `dig -x <host>`        |
+| Lookup local ip address                                                        | `hostname -i`          |
```

Comment:
Is it correct?

I got this error:
```
hostname: illegal option -- i
usage: hostname [-fs] [name-of-host]
```

Reply:
yes it is incorrect , I changed Name is "Display name of server" and Command is "hostname"